src/main/java/org/terasology/fluid/component/FluidContainerItemComponent.java

```diff
@@ -22,18 +22,31 @@
 import org.terasology.rendering.assets.texture.TextureRegionAsset;
 
 /**
- * @author Marcin Sciesinski <marcins78@gmail.com>
+ * This component indicates that an entity is a container capable of holding fluids.
  */
 public class FluidContainerItemComponent implements Component, ItemDifferentiating {
+    /** The type of the fluid it can contain */
     @Replicate
     public String fluidType;
+    /** The current volume of fluid in the container */
     public float volume;
+    /** The maximum volume of fluid that the container can contain */
     public float maxVolume;
+    /** The coordinate where the fluid 'filling' should start */
     public Vector2f fluidMinPerc;
+    /** The coordinate where the fluid 'filling' should end */
     public Vector2f fluidSizePerc;
+    /** The texture of the container when it is completely filled */
     public TextureRegionAsset<?> textureWithHole;
+    /** The texture of the container when it is empty */
     public TextureRegionAsset<?> emptyTexture;
 
+    /**
+     * Checks whether the fluid container's attributes are the same as those of a given object.
+     *
+     * @param o the Object to compare the fluid container to
```

Comment:
Every comment of the type "the X that does Y" needs to be changed to "The X that does Y." At one point you're actually doing the reverse - that should definitely be undone 😕